### PR TITLE
Added dynamic prop to tooltip component

### PR DESCRIPTION
**Pull request by vinesh-ghela**
Before submitting my PR to Code Review, I have made sure:

- [x] I have run the tests using `yarn run test`
- [x] I have exported any new files in index.js
- [x] My file supports Formik if will potentially be used in a form
- [ ] The PR name follows the convention `(#<ticket_number>) - <ticket_user_story>`
- [x] I have incremented the version in `package.json` if I want to publish a new version
- [x] If my change is breaking I have updated the minor version number(i.e. the middle one) and don't merge until I have the PR on Corazon ready

### CODE REVIEW

File: src/components/atoms/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -42,7 +43,7 @@ const Tooltip: React.FunctionComponent<IProps> = props => (
         className="visible-on-hover"
         delayHide={500}
         effect="solid"
-        type="success"
+        type={props.toolTipTheme}
```
Comment (alexwong-crc) [inline src/components/atoms/Tooltip/Tooltip.tsx:46]: we should pass more than just theme. We can pass in colour for the text, background and arrow. Can we pass those props too. Also, have you tried linking it to see if it works?

Comment (alexwong-crc) [reply src/components/atoms/Tooltip/Tooltip.tsx:46]: image

Comment (vinesh-ghela) [reply src/components/atoms/Tooltip/Tooltip.tsx]: Would i be able to get some advice on the linking please?

File: src/components/atoms/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
 interface IProps {
   showIcon?: boolean;
   id: string;
   className?: string;
   tooltipContent?: React.ReactNode;
+  toolTipTheme: Type;
+  toolTipArrowColor: string;
+  toolTipTextColor: string;
+  toolTipBackgroundColor: string;
+  toolTipBorder: boolean;
```
Comment (alexwong-crc) [inline src/components/atoms/Tooltip/Tooltip.tsx:15]: these should be optional.

File: src/components/atoms/Tooltip/Tooltip.tsx [hunk 3]
```diff
@@ -42,7 +49,11 @@ const Tooltip: React.FunctionComponent<IProps> = props => (
         className="visible-on-hover"
         delayHide={500}
         effect="solid"
-        type="success"
+        type= {props.toolTipTheme}
+        arrowColor={props.toolTipArrowColor}
+        textColor={props.toolTipTextColor}
+        border={props.toolTipBorder}
+        backgroundColor={props.toolTipBackgroundColor}
```
Comment (alexwong-crc) [inline src/components/atoms/Tooltip/Tooltip.tsx:56]: If they are not defined, does that affect it?

Comment (vinesh-ghela) [reply src/components/atoms/Tooltip/Tooltip.tsx:56]: its fine without it as long as `theme` is defined

Comment (alexwong-crc) [reply src/components/atoms/Tooltip/Tooltip.tsx:56]: Can we default theme to `success`

**Review by alexwong-crc — APPROVED** [review]

**Pull request merged**